docs/solarsystem/pds/pds.rst

```diff
@@ -0,0 +1,132 @@
+.. doctest-skip-all
+
+.. _astroquery.solarsystem.pds:
+
+***********************************************************************************
+PDS Planetary Ring Node Queries (`astroquery.solarsystem.pds`)
+***********************************************************************************
+
+Overview
+========
+
+
+The :class:`~astroquery.solarsystem.pds.RingNodeClass` class provides an
+interface to the ephemeris tools provided by the `NASA Planetary Data System's Ring Node System`_ hosted by SETI institute.
+
+Ephemeris
+-----------
+
+In order to query information for a specific Solar System body, a
+``RingNode`` object is instantiated and the :meth:`~astroquery.solarsystem.pds.RingNodeClass.ephemeris` method is called. The following example queries the
+ephemerides of the rings and small moons around Uranus as viewed from ALMA:
+
+.. code-block:: python
+
+   >>> from astroquery.solarsystem.pds import RingNode
+   >>> systemtable, bodytable, ringtable = RingNode.ephemeris(planet='Uranus',
+   ...                 obs_time='2024-05-08 22:39',
+   ...                 location = (-23.029 * u.deg, -67.755 * u.deg, 5000 * u.m))  # doctest: +SKIP
+   >>> print(ringtable)
+         ring  pericenter ascending node
+                  deg          deg
+       ------- ---------- --------------
+           Six    293.129           52.0
+          Five    109.438           81.1
+          Four    242.882           66.9
+         Alpha    184.498          253.9
+          Beta     287.66          299.2
+           Eta        0.0            0.0
+         Gamma     50.224            0.0
+         Delta        0.0            0.0
+        Lambda        0.0            0.0
+       Epsilon    298.022            0.0
+
+``planet`` must be one of ['mars', 'jupiter', 'uranus', 'saturn', 'neptune', 'pluto'] (case-insensitive)
+
+``obs_time`` is the datetime to query. Accepts a string in format 'YYYY-MM-DD HH:MM' (UTC assumed), or an ```astropy.Time`` object. If no obs_time is provided, the current time is used.
```

Comment:
use sphinx links, add them in single backticks with their correct namespace
```suggestion
``obs_time`` is the datetime to query. Accepts a string in format 'YYYY-MM-DD HH:MM' (UTC assumed), or an `~astropy.time.Time` object. If no obs_time is provided, the current time is used.
```

Reply:
This is a comment for here and below, too. Keep the double backticks for the parameter names, and anything else that you want to code highlight, but every public API should be linked.